docs/docusaurus/versioned_docs/version-0.18/oss/guides/connecting_to_your_data/datasource_configuration/sql_components/_note_optional_data_connector_keys.mdx

```diff
@@ -22,17 +22,17 @@
             <strong><code>included_tables</code>:</strong> A list that, if defined, will limit the inferred Data Assets to those tables that are included in the list.
         </li>
         <li>
-            <strong><code>skip_inapplicable_tables</code>:</strong>  A boolean value.  If <code>True</code>, tables that can't be successfully queried using sampling and splitter methods are excluded from inferred Data Assets. If <code>False</code>, the class will throw an error during initialization if any such tables are encountered.
+            <strong><code>skip_inapplicable_tables</code>:</strong>  A boolean value.  If <code>True</code>, tables that can't be successfully queried using sampling and partitioner methods are excluded from inferred Data Assets. If <code>False</code>, the class will throw an error during initialization if any such tables are encountered.
```

Comment:
I think we want to leave this `splitter` on the 0.18 branch.

Reply:
Oh good cal. I forgot we merged that work in. I'll take a look through here and see what needs to get reverted.